### PR TITLE
v1.7.4rc1 what's new

### PR DESCRIPTION
**Pull request by rhattersley**
A replacement for #1581 (as @pelson is mostly out of contact at the moment).

### CODE REVIEW

File: docs/iris/src/whatsnew/1.7.rst [hunk 1]
```diff
@@ -198,6 +198,17 @@ Bugs fixed in v1.7.3
 * Cubes that contain a pseudo-level coordinate can now be saved to PP.
 * Fixed a bug in the FieldsFile loader that prevented it always loading all available fields.
 
+Bugs fixed in v1.7.4
+^^^^^^^^^^^^^^^^^^^^
+* :meth:`Coord.guess_bounds` can now deal with circular coordinates.
+* :meth:`Coord.nearest_neighbour_index` can now work with descending bounds.
+* Passing `weights` to :meth:`Cube.rolling_window` no longer prevents other
+  other keyword arguments from being passed to the aggregator.
```
Comment (bjlittle) [inline docs/iris/src/whatsnew/1.7.rst]: Double `other` ...

**Comment by bjlittle** [comment]
@rhattersley Travis-CI is complaining that the example tests have failed ...

File: docs/iris/src/whatsnew/1.7.rst (hunk 1 above)
Comment (rhattersley) [inline docs/iris/src/whatsnew/1.7.rst]: That's spooky! :ghost: What are the odds of @pelson & me both making the same repeated-word mistake?

Comment (bjlittle) [inline docs/iris/src/whatsnew/1.7.rst]: I missed it on first reading ... maybe because it's split over two lines. Nice emoji btw!

**Comment by rhattersley** [comment]
> @rhattersley Travis-CI is complaining that the example tests have failed ...

Looks like a random time-out issue. I've restarted the job to see if it goes away.

File: lib/iris/__init__.py [hunk 2]
```diff
@@ -108,7 +108,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = '1.7.4-DEV'
+__version__ = '1.7.4rc1'
```
Comment (pelson) [inline lib/iris/__init__.py]: Really? I'd be comfortable for going straight to release. If there is something wrong with it, we've clearly misunderstood what a patch release means... and would therefore revert back to v1.7.3.

No comment necessary, just a point for future decisioning re versioning.

**Comment by pelson** [comment]
:+1: - thanks @rhattersley & @bjlittle!

File: lib/iris/__init__.py (hunk 2 above)
Comment (rhattersley) [inline lib/iris/__init__.py]: I'd still rather do a release candidate. I don't think that's at odds with the concept of a patch release at all.

**Pull request merged**